RealmSwift/Combine.swift

```diff
@@ -57,6 +57,18 @@ extension ObjectKeyIdentifiable {
     }
 }
 
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension Projection: Identifiable, Hashable {
```

Comment:
Unconditionally conforming to Identifiable means that users can't define their own `id` property. For normal objects we make this opt-in with ObjectKeyIdentifiable.